src/test/java/io/jenkins/plugins/artifact_manager_s3/S3AbstractTest.java

```diff
@@ -75,6 +78,9 @@ public static void live() {
     @Rule
     public LoggerRule loggerRule = new LoggerRule();
 
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
Probably could be avoided with some more careful refactoring ensuring that tests in `JCloudsVirtualFileTest` either never call `S3BlobStoreConfig.get()` or do not rely on its values, but will leave that to a future optimization.

Reply:
yep, I saw it there is some kind of duplication on configuration.